<!doctype html>
<html>
<head><link rel="stylesheet" type="text/css" href="stylesheet2.css"></head>
<body>



<div id="header"> <h1> ArunBio&#8482 </h1></div>
<div id="navbar"><ul>
<p>
<li><a class="button" href="file:///C:/Users/2031905/Desktop/Arunbio/index.html" >Home </a></li>
<li><a class="button" href="" >Services </a>
<ul>
    <li><a class="button" href="file:///C:/Users/%E0%B0%85%E0%B0%B0%E0%B1%81%E0%B0%A3%E0%B1%8D/Desktop/main/arunbio/Services%20databases.html" >Databases </a></li>
    <li><a class="button" href="file:///C:/Users/%E0%B0%85%E0%B0%B0%E0%B1%81%E0%B0%A3%E0%B1%8D/Desktop/main/arunbio/tools.html" >Tools </a></li>
<li><a class="button" href="https://www.youtube.com/channel/UC0BzWAnphrhYGMapUfvuhSQ" >Tutorials </a></li></li>
<li><a class="button" href="" >Reports </a></li>
</ul></li>

<li><a class="button" href="" >Blog</a></li>
<li><a class="button" href="file:///C:/Users/%E0%B0%85%E0%B0%B0%E0%B1%81%E0%B0%A3%E0%B1%8D/Desktop/main/arunbio/textbox.html" >Signup/login </a></li></p></ul>
 </div>
<div class="servicesmain"> <h3>Protein Databases</h3>
<fieldset>
<ol>
<li>
<p>UniProt
<a href = " https://www.uniprot.org/"  >link!
</a><br>
UniProt is a freely accessible database of protein sequence and functional information, many entries being derived from genome sequencing projects. It contains a large amount of information about the biological function of proteins derived from the research literature.
</p></li><li>
<p>DIP 
<a href = " https://dip.doe-mbi.ucla.edu/dip/Main.cgi"  >link!
</a><br>

The DIP database catalogs experimentally determined interactions between proteins. It combines information from a variety of sources to create a single, consistent set of protein-protein interactions. 
</p></li><li>
<p>InterPro 
<a href = " http://www.ebi.ac.uk/interpro/"  >link!
</a><br>
InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites. To classify proteins in this way, InterPro uses predictive models, known as signatures, provided by several different databases (referred to as member databases) that make up the InterPro consortium.
</p></li><li>
<p>MobiDB 
<a href = " https://mobidb.bio.unipd.it/ "  >link!
</a><br>
MobiDB provides information about intrinsically disordered regions (IDRs) and related features from various sources and prediction tools. Different levels of reliability and different features are reported as different and independent annotations
</p></li><li>
<p>neXtProt 
<a href = " https://www.nextprot.org/ "  >link!
</a><br>
neXtProt is a knowledgebase with expert curation and associated tools. It provides information on human proteins such as function, involvement in diseases, mRNA/protein expression, protein/protein interactions, PTMs, protein variations and their phenotypic effects.
</p></li><li>
<p>Pfam 
<a href = " https://pfam.xfam.org/ "  >link!
</a><br>
The Pfam database is a large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs).Pfam also generates higher-level groupings of related entries, known as clans. A clan is a collection of Pfam entries which are related by similarity of sequence, structure or profile-HMM.
</p></li><li>
<p>PRINTS 
<a href = " http://130.88.97.239/PRINTS/index.php/ "  >link!
</a><br>
In molecular biology, the PRINTS database is a collection of so-called "fingerprints": it provides both a detailed annotation resource for protein families, and a diagnostic tool for newly determined sequences. A fingerprint is a group of conserved motifs taken from a multiple sequence alignment - together, the motifs form a characteristic signature for the aligned protein family.
</p></li><li>
<p>PROSITE 
<a href = " https://prosite.expasy.org/ "  >link!
</a><br>
PROSITE is a protein database. It consists of entries describing the protein families, domains and functional sites as well as amino acid patterns and profiles in them. These are manually curated by a team of the Swiss Institute of Bioinformatics and tightly integrated into Swiss-Prot protein annotation.
</p></li><li>
<p>PIR 
<a href = " https://proteininformationresource.org/ "  >link!
</a><br>
The Protein Information Resource (PIR) is an integrated public resource of protein informatics that supports genomic and proteomic research and scientific discovery. PIR maintains the Protein Sequence Database (PSD), an annotated protein database containing over 283 000 sequences covering the entire taxonomic range.
</p></li><li>
<p>SUPERFAMILY 
<a href = " https://supfam.org/SUPERFAMILY/ "  >link!
</a><br>
SUPERFAMILY is a database of structural and functional annotation for all proteins and genomes. It classifies amino acid sequences into known structural domains, especially into SCOP superfamilies. Domains are functional, structural, and evolutionary units that form proteins.
</p></li><li>
<p>PDB 
<a href = " https://www.rcsb.org/ "  >link!
</a><br>
The Protein Data Bank (PDB) is a database for the three-dimensional structural data of large biological molecules, such as proteins and nucleic acids. The data, typically obtained by X-ray crystallography, NMR spectroscopy, or, increasingly, cryo-electron microscopy, and submitted by biologists and biochemists from around the world, are freely accessible on the Internet via the websites of its member organisations (PDBe, PDBj, RCSB, and BMRB)
</p></li><li>
<p>ModBase 
<a href = " https://modbase.compbio.ucsf.edu/modbase-cgi/index.cgi "  >link!
</a><br>
ModBase is a database of annotated comparative protein structure models, containing models for more than 3.8 million unique protein sequences. Models are created by the comparative modeling pipeline ModPipe which relies on the MODELLER program.
</p></li><li>
<p>PDBe-KB 
<a href = " https://www.ebi.ac.uk/pdbe/pdbe-kb/ "  >link!
</a><br>
PDBe-KB (Protein Data Bank in Europe - Knowledge Base) is a community-driven resource managed by the PDBe team, collating functional annotations and predictions for structure data in the PDB archive. PDBe-KB is a collaborative effort between PDBe and a diverse group of bioinformatics resources and research teams.
</p></li><li>
<p>SWISS-MODEL 
<a href = " https://swissmodel.expasy.org/ "  >link!
</a><br>
SWISS-MODEL is a structural bioinformatics web-server dedicated to homology modeling of 3D protein structures. Homology modeling is currently the most accurate method to generate reliable three-dimensional protein structure models and is routinely used in many practical applications. Homology (or comparative) modelling methods make use of experimental protein structures ("templates") to build models for evolutionary related proteins ("targets").
</p></li><li>
<p>BioGRID 
<a href = " https://thebiogrid.org/"  >link!
</a><br>
The BioGRID (Biological General Repository for Interaction Datasets, thebiogrid.org) is an open-access database resource that houses manually curated protein and genetic interactions from multiple species including yeast, worm, fly, mouse, and human.
</p></li>
</ol><fieldset></div>
<div id="footer">
<ul>
<p>
<li><a class="button" href="" >About ArunInc&#174 </a></li>
<li><a class="button" href="" >Careers </a></li>
<li><a class="button" href="" >Help </a></li>
<li><a class="button" href="" >Contact Us </a></li></p></ul> 
&copy; 2022 ArunBio&#8482 All Rights Reserved.</div>

</body>