<head> 
<link rel="stylesheet" type="text/css" href="stylesheet2.css"></head>
<body>
<div id="header"> <h1> ArunBio&#8482 </h1></div>
<div id="navbar"><ul>
<p>
<li><a class="button" href="file:///C:/Users/2031905/Desktop/Arunbio/index.html" >Home </a></li>
<li><a class="button" href="" >Services </a><ul>
    <li><a class="button" href="file:///C:/Users/%E0%B0%85%E0%B0%B0%E0%B1%81%E0%B0%A3%E0%B1%8D/Desktop/main/arunbio/Services%20databases.html" >Databases </a></li>
    <li><a class="button" href="file:///C:/Users/%E0%B0%85%E0%B0%B0%E0%B1%81%E0%B0%A3%E0%B1%8D/Desktop/main/arunbio/tools.html" >Tools </a></li>
<li><a class="button" href="https://www.youtube.com/channel/UC0BzWAnphrhYGMapUfvuhSQ" >Tutorials </a></li></li>
<li><a class="button" href="" >Reports </a></li>
</ul></li></li>
<li><a class="button" href="" >Blogs </a></li>
<li><a class="button" href="file:///C:/Users/%E0%B0%85%E0%B0%B0%E0%B1%81%E0%B0%A3%E0%B1%8D/Desktop/main/arunbio/textbox.html" >Signup/login </a></li></p></ul>
 </div>


<div class="servicesmain"> <h3>Tools</h3>

<div id="nav"><ul><p>
<li> <a class="button" href="#translation"> Translation</a></li>
<li> <a class="button" href="#reversetranslation"> Reverse Translation </a></li>
<li> <a class="button" href="#homologymodelling"> Homology Modelling </a></li>
<li> <a class="button" href="#multiplesequencealignment"> Multiple Sequence Alignment </a></li>
<li><a class="button" href="#pairwisesequencealignment"> Pairwise Sequence Alignment </a></li>
</br><li><a class="button" href="#primerdesign"> Primer Design </a></li>
<li><a class="button" href="#primerannealing"> Primer Annealing </a></li></p>
</ul> </div> </br> </br> </br></br>
<h2 class="sideheadings"> <a name="translation">Translation</a></h2>
<p>Translation is the process in which ribosomes in the cytoplasm or endoplasmic reticulum synthesize proteins after the process of transcription of DNA to RNA in the cell's nucleus. The entire process is called gene expression.</p><fieldset><ol>
<li>
<p>Expasy 
<a href = " https://web.expasy.org/translate/"  >link!
</a><br>
Expasy Translate is a tool which allows the translation of a nucleotide (DNA/RNA) sequence to a protein sequence.
</p></li><li>
<p>EMBOSS Transeq
<a href = "https://www.ebi.ac.uk/Tools/st/emboss_transeq/ "  >link!
</a><br>
EMBOSS Transeq translates nucleic acid sequences to their corresponding peptide sequences. It can translate to the three forward and three reverse frames, and output multiple frame translations at once.
</p></li><li>
<p> EMBOSS Sixpack 
<a href = "https://www.ebi.ac.uk/Tools/st/emboss_sixpack/ "  >link!
</a><br>
Sixpack reads a DNA sequence and outputs the three forward and (optionally) three reverse translations in a visual manner.
</p></li>
</ol></fieldset>


<h2 class="sideheadings"> <a name="reversetranslation"> Reverse translation</h2></p>
<p> Reverse Translation  is a process where a protein sequence is analysed to get nearest possible nucleic acid sequence</p><fieldset><ol>
<li>

<p>EMBOSS Backtranseq 
<a href = " https://www.ebi.ac.uk/Tools/st/emboss_backtranseq/ "  >link!
</a><br>
EMBOSS Backtranseq reads a protein sequence and writes the nucleic acid sequence it is most likely to have come from.
</p></li><li>
<p>EMBOSS Backtranambig 
<a href = "https://www.ebi.ac.uk/Tools/st/emboss_backtranambig/ "  >link!
</a><br>
 Backtranambig reads a protein sequence and writes the nucleic acid sequence it could have come from. It does this by using nucleotide ambiguity codes that represent all possible codons for each amino acid.
</p></li><li>
<p>SMS Reverse Translate
<a href = " http://www.bioinformatics.org/sms2/rev_trans.html "  >link!
</a><br>
Reverse Translate accepts a protein sequence as input and uses a codon usage table to generate a DNA sequence representing the most likely non-degenerate coding sequence. A consensus sequence derived from all the possible codons for each amino acid is also returned.
</p></li>
</ol></fieldset>

<h2 class="sideheadings"> <a name="homologymodelling"> Homology Modelling</h2></p>
<p>Homology modeling, also known as comparative modeling of protein, refers to constructing an atomic-resolution model of the " target " protein from its amino acid sequence and an experimental three-dimensional structure of a related homologous protein (the " template ").</p><fieldset><ol><li>
<p> LOMETS
<a href = "https://zhanggroup.org/LOMETS/ "  >link!
</a><br>
LOMETS (Local Meta-Threading Server, version 3) is a new generation of meta-server approach to template-based protein structure prediction and structure-based function annotation, which integrates multiple deep learning-based threading methods (CEthreader, DisCovER, EigenThreader, Hybrid-CEthreader, MapAlign) and state-of-the-art profile-based programs (FFAS3D, HHpred, HHsearch, MRFsearch, MUSTER, SparksX). 
</p></li><li>
<p>CPHmodels 
<a href = " https://services.healthtech.dtu.dk/service.php?CPHmodels-3.2 "  >link!
</a><br>
CPHmodels 3.2 is a protein homology modeling server. The template recognition is based on profile-profile alignment guided by secondary structure and exposure predictions.
</p></li>
<li>
<p>Phyre<sup>2</sup> 
<a href = " http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index "  >link!
</a><br>

Phyre and Phyre2 (Protein Homology/AnalogY Recognition Engine; pronounced as 'fire') are free web-based services for protein structure prediction. It is able to regularly generate reliable protein models.
</p></li>


</ol> </fieldset>

<h2 class="sideheadings"> <a name="multipleSequencealignment"> Multiple Sequence Alignment</h2></p>
<p>Multiple Sequence Alignment (MSA) is generally the alignment of three or more biological sequences (protein or nucleic acid) of similar length. From the output, homology can be inferred and the evolutionary relationships between the sequences studied.</p>
<fieldset><ol>
<li>
<p> Clustal Omega
<a href = "https://www.ebi.ac.uk/Tools/msa/clustalo/ "  >link!
</a><br>
Clustal Omega is a new multiple sequence alignment program that uses seeded guide trees and HMM profile-profile techniques to generate alignments between three or more sequences
</p></li><li>
<p>EMBOSS Cons 
<a href = " https://www.ebi.ac.uk/Tools/msa/emboss_cons/ "  >link!
</a><br>
EMBOSS Cons creates a consensus sequence from a multiple alignment (MSF, PIR, CLUSTAL, etc).
</p></li>
<li>
<p> Kalign 
<a href = "https://www.ebi.ac.uk/Tools/msa/kalign/ "  >link!
</a><br>
A fast and accurate multiple sequence alignment algorithm. 
</p></li><li>
<p>MAFFT
<a href = " https://www.ebi.ac.uk/Tools/msa/mafft/ "  >link!
</a><br>
MAFFT (Multiple Alignment using Fast Fourier Transform) is a high speed multiple sequence alignment program.
</p></li>
<li>
<p> MUSCLE
<a href = "https://www.ebi.ac.uk/Tools/msa/muscle/ "  >link!
</a><br>
MUSCLE stands for MUltiple Sequence Comparison by Log- Expectation. MUSCLE is claimed to achieve both better average accuracy and better speed than ClustalW2 or T-Coffee, depending on the chosen options. 
</p></li><li>
<p>T-Coffee
<a href = " https://www.ebi.ac.uk/Tools/msa/tcoffee/ "  >link!
</a><br>
T-Coffee is a multiple sequence alignment program. Its main characteristic is that it will allow you to combine results obtained with several alignment methods.
</p></li>
</ol></fieldset>

<h2 class="sideheadings"> <a name="pairwisesequencealignment"> Pairwise Alignment </h2></p>
<p> Pairwise Sequence Alignment is used to identify regions of similarity that may indicate functional, structural and/or evolutionary relationships between two biological sequences (protein or nucleic acid)</p>
<fieldset><ol>
<p><b> Global Alignment</b></p>
<p> Global alignment tools create an end-to-end alignment of the sequences to be aligned.</p>
<li>
<p> EMBOSS Needle 
<a href = "https://www.ebi.ac.uk/Tools/psa/emboss_needle/ "  >link!
</a><br>
EMBOSS Needle reads two input sequences and writes their optimal global sequence alignment to file.
</p></li><li>
<p>EMBOSS Stretcher 
<a href = " https://www.ebi.ac.uk/Tools/psa/emboss_stretcher/ "  >link!
</a><br>
EMBOSS Stretcher calculates an optimal global alignment of two sequences using a modification of the classic dynamic programming algorithm which uses linear space.
</p></li>
<li>
<p> GGSEARCH2SEQ 
<a href = "https://www.ebi.ac.uk/Tools/psa/ggsearch2seq/ "  >link!
</a><br>
GGSEARCH2SEQ finds an optimal global alignment using the Needleman-Wunsch algorithm.
</p></li></ol></fieldset>

<p><b>Local Alignment</b></p>
<fieldset><ol>
<p>Local alignment tools find one, or more, alignments describing the most similar region(s) within the sequences to be aligned. They are can align protein and nucleotide sequences.</p>

<li>
<p>Water 
<a href = " https://www.ebi.ac.uk/Tools/psa/emboss_water/ "  >link!
</a><br>
EMBOSS Water uses the Smith-Waterman algorithm (modified for speed enhancements) to calculate the local alignment of two sequences.
</p></li>
<li>
<p> Matcher
<a href = "https://www.ebi.ac.uk/Tools/psa/emboss_matcher/ "  >link!
</a><br>
EMBOSS Matcher identifies local similarities in two input sequences using a rigorous algorithm based on Bill Pearson's lalign application, version 2.0u4 (Feb. 1996).
</p></li><li>
<p>LAlign
<a href = " https://www.ebi.ac.uk/Tools/psa/lalign/ "  >link!
</a><br>
LALIGN finds internal duplications by calculating non-intersecting local alignments of protein or nucleotide sequences.
</p></li>
<li>
<p>SSEARCH2SEQ
<a href = " https://www.ebi.ac.uk/Tools/psa/ssearch2seq/ "  >link!
</a><br>
SSEARCH2SEQ finds an optimal local alignment using the Smith-Waterman algorithm.
</p></li>
<li>
<p>GeneWise
<a href = " https://www.ebi.ac.uk/Tools/psa/genewise/ "  >link!
</a><br>

GeneWise compares a protein sequence to a genomic DNA sequence, allowing for introns and frameshifting errors.
</p></li></ol></fieldset>


<h2 class="sideheadings"> <a name="primerdesign">Primer Design</a></h2>
<fieldset><ol>
<li>
<p> Primer-BLAST
<a href = " https://www.ncbi.nlm.nih.gov/tools/primer-blast/ "  >link!
</a><br>
A tool for finding specific primers.
</p></li>
<li>
<p> Primer3web
<a href = "https://primer3.ut.ee/ "  >link!
</a>
</p></li><li>
<p> Primer3Plus
<a href = " https://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi/ "  >link!
</a>

</p></li>
<li>
<p>GenScript Real-time PCR (TaqMan) Primer Design
<a href = " https://www.genscript.com/tools/real-time-pcr-taqman-primer-design-tool "  >link!
</a>
/p></li>
<li>
<p> AutoPrime
<a href = " http://www.autoprime.de/AutoPrimeWeb "  >link!
</a>
</p></li>
</ol></fieldset>

<h2 class="sideheadings"> <a name="primerannealing">Primer Annealing</a></h2>
 <h3> COMING SOON!!! </h3>
<fieldset></div>
<div id="footer">
<ul>
<p>
<li><a class="button" href="" >About ArunInc&#174 </a></li>
<li><a class="button" href="" >Careers </a></li>
<li><a class="button" href="" >Help </a></li>
<li><a class="button" href="" >Contact Us </a></li></p></ul> 
&copy; 2022 ArunBio&#8482 All Rights Reserved.</div>

</body>